api/src/migrations/extract-translations.js

```diff
@@ -31,16 +31,9 @@ const mergeTranslations = (settings, docs) => {
     setting.translations.forEach(translation => {
       const doc = _.findWhere(docs, { code: translation.locale });
       if (doc) {
-        if (_.has(doc, 'values')) {
-          if (!doc.values[setting.key] || translation.content !== translation.default) {
-            // only update the doc if it was changed from the default
-            doc.values[setting.key] = translation.content;
-          }
-        } else { 
-          if (!doc.generic[setting.key] || translation.content !== translation.default) {
-            // only update the doc if it was changed from the default
-            doc.generic[setting.key] = translation.content;
-          }
+        if (!doc.values[setting.key] || translation.content !== translation.default) {
+          // only update the doc if it was changed from the default
+          doc.values[setting.key] = translation.content;
```

Comment:
Part of Stefan's WIP with Peter. 
Do you think we should actually keep the values/generic check?

Reply:
Yes I think we need to based on the order this will run. What do you think?